packages/main/cypress/specs/BusyIndicator.cy.ts

```diff
@@ -0,0 +1,186 @@
+import { html } from "lit";
+import "../../src/BusyIndicator.js";
+import "../../src/Button.js";
+import "../../src/List.js";
+import "../../src/ListItem.js";
+import "../../src/Dialog.js";
+
+const busyIndicator = html`<ui5-busy-indicator size="M" active id="indicator1"></ui5-busy-indicator>`;
+
+describe("BusyIndicator general interaction", () => {
+	it("tests event propagation", () => {
+		cy.mount(html`
+            <ui5-busy-indicator id="bInticator" active delay="0" size="M">
+                <ui5-button id="enblBtn"> Enabled< /ui5-button>
+                <ui5-button> Enabled </ui5-button>
+            </ui5-busy-indicator>
+        `);
+
+		cy.get("#enblBtn")
+			.shadow()
+			.find("button")
+			.as("button");
+
+		cy.get("#enblBtn").then($button => {
+			$button.get(0).addEventListener("click", cy.stub().as("clicked"));
+		});
+
+		cy.get("#enblBtn").realClick();
+		cy.get("@clicked")
+			.should("have.callCount", 0);
+	});
+
+	it("test activation", () => {
+		cy.mount(html` <ui5-busy-indicator id="busy-container" delay="200" size="M" text="Loading">
+            <ui5-list id="fetch-list" no-data-text="No Data" header-text="Available Items"></ui5-list>
+        </ui5-busy-indicator>`);
+
+		cy.get("#busy-container")
+			.should("not.have.attr", "_is-busy");
+
+		// eslint-disable-next-line cypress/no-unnecessary-waiting
+		cy.get("#busy-container")
+			.invoke("attr", "active", "")
+			.wait(300)
+			.should("have.attr", "_is-busy");
+	});
+
+	it("tests focus handling", () => {
+		cy.mount(busyIndicator);
+
+		cy.get("#indicator1")
+			.shadow()
+			.find(".ui5-busy-indicator-busy-area")
+			.realClick();
+
+		cy.get("#indicator1")
+			.should("be.focused");
+	});
+
+	it("tests internal focused element attributes", () => {
+		cy.mount(busyIndicator);
+
+		cy.get("#indicator1")
+			.shadow()
+			.find(".ui5-busy-indicator-busy-area")
+			.should("have.attr", "role", "progressbar")
+			.should("have.attr", "tabindex", "0")
+			.should("have.attr", "aria-valuemin", "0")
+			.should("have.attr", "aria-valuemax", "100")
+			.should("have.attr", "aria-valuetext", "Busy");
+	});
+
+	it("tests internal focused element attributes", () => {
```

Comment:
test name should be "tests content is not reachable with keyboard when active in both directions"